### PR TITLE
Issue113: Refactoring InformationSheetContent file: 100% DONE

### PR DESCRIPTION
**Pull request by dearpriyankasa**

### CODE REVIEW

**Comment by drewlearns** [comment]
Your review app has been deployed and is available at 'http://20.85.187.248:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by drewlearns** [comment]
Your review app has been deployed and is available at 'http://20.85.187.248:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/pages/InformationSheetContent/PlantingAndGrowthWindows/PlantingAndGrowthWindows.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { AccordionDetails, Typography } from '@mui/material';
+import { ExpandMore, FiberManualRecord } from '@mui/icons-material';
+import moment from 'moment-timezone';
+import CropSelectorCalendarView from '../../../components/CropSelectorCalendarView/CropSelectorCalendarView';
+import MonthDayString from './getMonthDayString';
```
Comment (mikahpinegar) [inline src/pages/InformationSheetContent/PlantingAndGrowthWindows/PlantingAndGrowthWindows.js:6]: this should be getMonthDayString

**Review by mikahpinegar — CHANGES_REQUESTED** [review]

File: src/pages/InformationSheetContent/PlantingAndGrowthWindows/PlantingAndGrowthWindows.js (hunk 1 above)
Comment (dearpriyankasa) [reply src/pages/InformationSheetContent/PlantingAndGrowthWindows/PlantingAndGrowthWindows.js]: Done

**Comment by drewlearns** [comment]
Your review app has been deployed and is available at 'http://20.85.187.248:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by drewlearns** [comment]
Your review app has been deployed and is available at 'http://20.85.187.248:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by mikahpinegar — APPROVED** [review]

**Pull request merged**